Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:16.04
+MAINTAINER "Greg Way" <gregway@mail.med.upenn.edu>
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+            libcurl4-openssl-dev \
+            libxml2-dev \
+            ed \
+            git \
+            vim \
+            curl \
+            r-base \
+            r-base-dev \
+            r-recommended
+
+RUN git clone https://github.com/greenelab/gbm_immune_validation.git
```

Comment:
So the code is in the Docker image... but what code does Drone use: the Docker code or the committed code that triggers the CI build?